lib/index.js

```diff
@@ -94,7 +97,7 @@ function makeObject(info) {
         var jsPropertyName = propertyName.replace(/-/g, '_');
 
         Object.defineProperty(constructor.prototype, jsPropertyName, {
-            enumerable: true,
+            configurable: true,
```

Comment:
quick recap ... there are two kind of descriptors in JS, data and accessors

**data flags**
configurable, enumerable, writable, value


**accessor flags**
configurable, enumerable, setter, getter

Both configurable and enumerable are by default false but in native "classes" are configurable and non enumerable.

Data descriptor are writable and have a value in native too, simplifying polyfill wrapping.